static/app/views/settings/organizationTeams/teamMembers.spec.jsx

```diff
@@ -17,7 +17,11 @@ describe('TeamMembers', function () {
   let createMock;
 
   const organization = TestStubs.Organization();
-  const team = TestStubs.Team();
+  const team = TestStubs.Team({
+    flags: {
+      'idp:provisioned': false,
+    },
+  });
```

Comment:
Might make more sense to update the stub with 

```
    flags: {
      'idp:provisioned': false,
    },
```

so it can be consistent across all tests.

This can be a separate PR.